src/components/ShareTrajectoryModal/EmbedPreview.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import CustomModal from "../CustomModal";
+
+import styles from "./style.css";
+
+interface EmbedPreviewProps {
+    embedSnippet: string;
+    height: number;
+    width: number;
+    preview: boolean;
+    setPreview: (value: boolean) => void;
+}
+
+const EmbedPreview: React.FC<EmbedPreviewProps> = ({
+    embedSnippet,
+    height,
+    width,
+    preview,
+    setPreview,
+}) => {
+    document.documentElement.style.setProperty(
+        "--embed-preview-modal-width",
+        `${width + 50}px`
+    );
+    document.documentElement.style.setProperty(
+        "--embed-preview-modal-height",
+        `${height + 112}px`
+    );
```

Comment:
Need a way to get these changing values into the .antd selector, am I missing an easier way?

Reply:
does setting a padding or margin not work?